backend/src-common/src/main/java/org/eclipse/sw360/datahandler/db/DatabaseHandlerUtil.java

```diff
@@ -795,4 +808,52 @@ public static void saveAttachmentInFileSystem(AttachmentConnector attachmentConn
                             attachmentContentId, att.getFilename());
                 });
     }
-}
\ No newline at end of file
+
+    public static RequestStatus deleteOldAttachmentFromFileSystem() {
+        String targetDir = ATTACHMENT_STORE_FILE_SYSTEM_LOCATION;
```

Comment:
Maybe it is not needed to create a separate variable instead `ATTACHMENT_STORE_FILE_SYSTEM_LOCATION` could be used directly.
`String targetDir = ATTACHMENT_STORE_FILE_SYSTEM_LOCATION;`